fluent-plugin-prometheus.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = "fluent-plugin-prometheus"
-  spec.version       = "2.1.0"
+  spec.version       = "2.1.1"
```

Comment:
Could you remove this version change from this pull request?
It will be controlled by us.
In addition, this pull request adding a new feature, so the minor version should be incremented instead of the patch version.

Reply:
Yah sure
I will revert this file